joss/paper.md

```diff
@@ -40,6 +40,27 @@ The Julia package `Copulas.jl` brings most standard copula-related features into
 The R package `copula` [@r_copula_citation1; @r_copula_citation2; @r_copula_citation3; @r_copula_citation4] is the gold standard when it comes to sampling, estimating, or simply working around dependence structures. However, in other languages, the available tools are not as developed and/or not as recognized. We bridge the gap in the Julian ecosystem with this Julia-native implementation. Due to the very flexible type system in Julia, our code expressiveness and tidiness will increase its usability and maintainability in the long-run. Type-stability allows sampling in arbitrary precision without requiring more code, and Julia's multiple dispatch yields most of the below-described applications.
 
 There are competing packages in Julia, such as [`BivariateCopulas.jl`](https://github.com/AnderGray/BivariateCopulas.jl) [@BivariateCopulas] which only deals with a few models in bivariate settings but has very nice graphs, or [`DatagenCopulaBased.jl`](https://github.com/iitis/DatagenCopulaBased.jl) [@DatagenCopulaBased_1; @DatagenCopulaBased_2; @DatagenCopulaBased_3; @DatagenCopulaBased_4], which only provides sampling and does not have exactly the same models as `Copulas.jl`. While not fully covering out both of these package's functionality (mostly because the three projects chose different copulas to implement), `Copulas.jl` is clearly the most fully featured, and brings, as a key feature, the compliance with the broader ecosystem.
+# Comparison of other packages with copulas.jl
+According to the package documentation, it is possible to summarize some of the most important functionalities of each package and those most needed by the community.
+## Functionality
+The following table shows some characteristics that differentiate each package.
+| Characteristic                                | Copulas.jl         | DatagenCopulaBased.jl         | BivariateCopulas.jl          |
+|-----------------------------------------------|--------------------|--------------------|--------------------|
+| Every Archimedean Copula sampling     | Yes                 | No                 | No                 |

```

Comment:
You could add here a line "Classic Bivariate copulas sampling" with Yes/Yes/Yes. 

Also a line "Obscure bivariate copulas sampling" with "Yes/No/No ?" ^^

Maybe the first column could be refactored to : 
Sampling
    - Classic Bivariate Copulas  Yes Yes Yes 
    - Obscure Bivariate Copulas Yes No No
    - Archimedean copulas All / Classic Only / Classic Only
    - Multivariate Copulas Yes Yes No (You exchanged the two i think)
    - Archimedean Chains (this is the real name of "nested" : No No Yes
Fitting   No No Yes
Plotting No No Yes
Dependence metrics Partial / ?  / ? (check)